provisionersdk/proto/provisioner.proto

```diff
@@ -101,6 +101,8 @@ message Preset {
   repeated PresetParameter parameters = 2;
   Prebuild prebuild = 3;
   bool default = 4;
+  string description = 5;
+  string icon = 6;
```

Comment:
We need to increment `proto.CurrentMinor` due to the addition of new fields here.